src/pages/[bookId].js

```diff
@@ -0,0 +1,48 @@
+import Head from 'next/head';
+import {useRouter} from 'next/router';
+import styled from 'styled-components';
+
+import Layout from '../components/Layout';
+import Title from '../components/StyledTitle';
+
+export default function Detailpage({books}) {
+	const router = useRouter();
+	const bookId = router.query.bookId;
+
+	const book = books.find(event => event._id === bookId);
+	console.log(bookId);
+	console.log(books);
+
+	return (
+		<Layout>
+			<Head>
+				<title key="title">Detail page</title>
+				<meta key="description" name="description" content="bookDetails" />
+			</Head>
+			{book && (
+				<>
+					<Title>{book.title}</Title>
+					<DetailCard>
+						<p>Author: {book.author}</p>
+						<p>Rating: {book.rating}/10</p>
+						<h3>Reading Notes</h3>
+						<p>{book.readingNotes}</p>
+					</DetailCard>
+				</>
+			)}
+		</Layout>
+	);
+}
+
+const DetailCard = styled.span`
```

Comment:
I would prefer to work with a list. The `DetailCard` could be the `ul` element and the `p` and `h3` elements can be the `li` elements. Please think about it. I would not use a `span`.